app/dao/provider_details_dao.py

```diff
@@ -30,13 +28,10 @@ def get_provider_details_by_identifier(identifier):
 
 
 def get_alternative_sms_provider(identifier):
-    alternate_provider = None
-    if identifier == SNS_PROVIDER:
-        alternate_provider = PINPOINT_PROVIDER
-    elif identifier == PINPOINT_PROVIDER:
-        alternate_provider = SNS_PROVIDER
-
-    return ProviderDetails.query.filter_by(identifier=alternate_provider).one()
+    # We currently run with a single SMS provider (SNS)
+    # so this method is not implemented and does not
+    # switch providers.
+    return get_provider_details_by_identifier(identifier)
```

Comment:
Rewrote the method to do nothing for the time being: returns the same provider as the current one. This does not perform an actual provider switch.

Reply:
Do we want to do anything with the switching logic? In an exception condition, we end up running a bunch of logic to switch from:
https://github.com/cds-snc/notification-api/blob/a35f8fe77bc9f5a7fd8b32f567ee720ce241d091/app/delivery/send_to_providers.py#L82

Which may be redundant and set off an intended chain of events.

Reply:
Thanks for taking a good look at this, appreciated that you went down the road.

I wanted to leave as many things untouched for this path because I that the logic is good and we'll need to it. I think that there is a cost of deleting it now and introducing it again.

If you follow the underlying method calls https://github.com/cds-snc/notification-api/blob/a35f8fe77bc9f5a7fd8b32f567ee720ce241d091/app/dao/provider_details_dao.py#L66
and then https://github.com/cds-snc/notification-api/blob/a35f8fe77bc9f5a7fd8b32f567ee720ce241d091/app/provider_details/switch_providers.py#L22
https://github.com/cds-snc/notification-api/blob/a35f8fe77bc9f5a7fd8b32f567ee720ce241d091/app/provider_details/switch_providers.py#L39

So nothing will happen for real but we'll still log messages and add new rows to the providers' history table, which I think is important for logging purposes.